### ISSUE TITLE
some (all?) BYG ores have no use

### ISSUE THREAD

**Issue by s0urdough** [NillerMedDild/Enigmatica6#3398]
### Modpack Version

E6 0.5.10

### Describe your issue.

Certain BYG ores don't seem to be usable in any way. They can't be smelted, they can't be used with any machines, they have no drops aside from the blocks themselves, and they can't be used with the stonecutter to change them into another type of ore. The specific ones I've tried are: 
- Brimstone Nether Quartz Ore _byg:brimstone_nether_quartz_ore_
- Brimstone Nether Gold Ore _byg:brimstone_nether_gold_ore_
- Blue Nether Quartz Ore _byg:blue_nether_quartz_ore_
- Blue Nether Gold Ore _byg:blue_nether_gold_ore_

All of these blocks seem to be missing the _forge:ores/gold_ tag. They (and all BYG ores) also don't show up in JEI, and as a result I can't check to see if other BYG ores have the same problem.

### Have you modified the modpack?

Yes

### User Modifications

We've added RandomPatches to help with users timing out on login.

### Did the issue happen in singleplayer or on a server?

Server

**Comment by s0urdough** [comment]
Ah, my mistake about the addition of RandomPatches, I only changed the timeout settings in its config.

And the ores were obtained using an RFTools Builder Quarry, from the nether. I think the world was originally generated in an older version, but I'm not sure which.

**Comment by NillerMedDild** [comment]
The BYG Nether is disabled, you're not supposed to be able to obtain these ores in any way :)

Did you manually enable the BYG Nether or?

**Comment by s0urdough** [comment]
> Did you manually enable the BYG Nether or?

I don't think so but it's possible, where would I check? AFAIK the world was created several months ago and then updated recently. Was the BYG nether always disabled?

**Comment by NillerMedDild** [comment]
> Still, seems odd the blocks don't have the proper tags to work. Even if they shouldn't be obtainable there shouldn't be any harm in allowing them to be processed as normal.

The materials are disabled completely, I don't really like them. They only exist to make armor iirc.
